endpoints.json

```diff
@@ -156,5 +156,24 @@
   }
 ]
 }
+  },
+
+  "GET /api/articles?sort_by=title&order=asc": {
+    "description": "200: sorts articles by a valid column in ascending order",
+    "queries": ["title", "asc"],
+    "exampleResponse": {
+      "articles": [
+        {
+          "author": "butter_bridge",
+          "title": "Living in the shadow of a great man",
+          "article_id": 1,
+          "topic": "mitch",
+          "created_at": "2020-07-09T20:11:00.000Z",
+          "votes": 100,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          "comment_count": 11
+        }
+      ]
+    }
```

Comment:
This shouldnt be a seperate entry here and should just be part of the original with the queries array having the name of the queries as `["sort_by", "order"]`